ComponentKit/Components/CKButtonComponent.mm

```diff
@@ -95,6 +95,10 @@ + (instancetype)newWithTitles:(const std::unordered_map<UIControlState, NSString
                    attributes:(const CKViewComponentAttributeValueMap &)passedAttributes
    accessibilityConfiguration:(CKButtonComponentAccessibilityConfiguration)accessibilityConfiguration
 {
+  std::unordered_map<UIControlState, NSString *> titles = titlesParam.take();
+  std::unordered_map<UIControlState, UIColor *> titleColors = titleColorsParam.take();
+  std::unordered_map<UIControlState, UIImage *> images = imagesParam.take();
+  std::unordered_map<UIControlState, UIImage *> backgroundImages = backgroundImagesParam.take();
```

Comment:
I don't mind the typing, but I can certainly do that if you think it improves readability.